src/OVAL/probes/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ if(ENABLE_PROBES)
 
 		add_oscap_probe(probe_family "independent/family.c")
 
+		if(NOT WIN32)
 		add_oscap_probe(probe_textfilecontent "independent/textfilecontent.c")
```

Comment:
What to do about indentation here? I agree that adding a tab to all lines until `endif()` could cause merge conflicts, but on the other hand, there are not multiple branches with actively developed CMake content. So I suggest to increase indentation of the if block.

Reply:
OK